Dockerfile

```diff
@@ -9,5 +9,5 @@ ADD scm-source.json /scm-source.json
 EXPOSE 8080
 
 # run the server when a container based on this image is being run
-ENTRYPOINT java -jar -Djava.security.egd=file:/dev/./urandom nakadi.jar
+ENTRYPOINT java $JAVA_OPTS -jar -Djava.security.egd=file:/dev/./urandom nakadi.jar
```

Comment:
@v-stepanov My point was only about the order
@rcillo We have two different calls for getting different variables:
```
        System.getenv("name") // gets system dependent environment variable
        System.getProperty("name") // gets system property passed using -D
```
Probably tool, that you provided link to tries to get variables from different places.
For our purposes: we need -X and other switches, that can't be passed using environment vairables.